### PR TITLE
Add transparent decompression (funpack) support

### PR DESCRIPTION
**Pull request by irasnyd**
Add two new Django views to return files from the backend storage (AWS
S3 Bucket). These views will be used to serve files to the NGINX mod_zip
module, so that users can choose between compressed or decompressed
files as they download a ZIP archive.

The Archive Client will need to be extended to add support to select
between these two types of downloads.

Educational users have expressed difficulty with handling decompression
of FITS files. This should help improve their experience.

### CODE REVIEW

**Comment by markBowman** [comment]
What's the status of this PR?

**Comment by dmcollom** [comment]
> What's the status of this PR?

The status is logged in a comment in [this story](https://www.pivotaltracker.com/story/show/171163728), but the short of it is that the internal-only funpack endpoint doesn't appear to be working, and the story to complete the work has never made it into a sprint. The file that results from the internal funpack endpoint is corrupt in some way and the work wasn't completed in the sprint in which the original story was included.

**Comment by markBowman** [comment]
> > What's the status of this PR?
> 
> The status is logged in a comment in [this story](https://www.pivotaltracker.com/story/show/171163728), but the short of it is that the internal-only funpack endpoint doesn't appear to be working, and the story to complete the work has never made it into a sprint. The file that results from the internal funpack endpoint is corrupt in some way and the work wasn't completed in the sprint in which the original story was included.

Thanks. This had completely dropped off my stack. I've moved the story from the icebox to the backlog, as a reminder.

**Comment by dmcollom** [comment]
This should finally be ready for review. It's been tested in dev downloading 100 files at once for both the compressed and uncompressed endpoints, and required a memory bump for the uncompressed download that will be reflected in the PR in helm-charts.

File: archive/frames/views.py [hunk 1]
```diff
@@ -109,6 +112,7 @@ def headers(self, request, pk=None):
     @xframe_options_exempt
     @list_route(methods=['post'], permission_classes=[AllowAny])
     def zip(self, request):
+        logger.info(msg='Reached zip endpoint')
```
Comment (jnation3406) [inline archive/frames/views.py:115]: im guessing we don't want this anymore

**Review by jnation3406 — APPROVED** [review]

**Pull request merged**